plugins/influxdb/README.md

```diff
@@ -1,22 +1,78 @@
 # InfluxDB plugin
 
-Provides an output to InfluxDB 2.
+Provides an output to InfluxDB v2.
 
-## Config options
+## Requirements
 
-- host: InfluxDB server URL, for example `http://localhost:8086`. You can also use `https`.
-- token: your authentication token
-- org: organization to write the measurements to
-- bucket: bucket to write the measurements to
-- attribute_as: how to serialize the Alumet attributes. This can be either `"field"` or `"tag"`.
-- attribute_as_tags (optional): always serialize the given list of attributes as InfluxDB tags
-- attribute_as_fields (optional): always serialize the given list of attributes as InfluxDB fields
+- Write access to a running instance of InfluxDB v2.
 
-## Attribute serialization
+## Configuration
+
+Here is an example of how to configure this plugin. Put the following in the configuration file of the Alumet agent (usually alumet-config.toml).
+
+```toml
+[plugins.influxdb]
+# Address of the host where InfluxDB is running
+host = "http://localhost:8086"
+# Token to write on the database
+token = "FILL ME"
+# Organisation and bucket where to write data
+org = "FILL ME"
+bucket = "FILL ME"
+# By default, serialize all Alumet attributes as fields. This can be either `"field"` or `"tag".
+attributes_as = "field"
+# Always serialize the given list of attributes as InfluxDB tags
+attributes_as_tags = [""]
+# Always serialize the given list of attributes as InfluxDB fields
+attributes_as_fields = [""]
+```
+
+## More information
+
+### Attribute serialization
 
 InfluxDB does not have "attributes", but "tags" (which are indexed and can only hold strings) and "fields" (which are not indexed and can hold strings, integers, floats and booleans).
+For tags, Alumet will automatically serialize the values to strings.
+
+By changing the config options, you can choose which attributes translate to tags and which ones translate to fields.
+
+For example, depending on the config, the same alumet point will lead to different influxdb point. Here is it's json representation.
```

Comment:
What is the ìt`in the last sentence? The "alumet point" or the "influxdb point"?